Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1102,6 +1102,49 @@ theorem eLpNorm'_le_nnreal_smul_eLpNorm'_of_ae_le_mul {f : α → F} {g : α →
   simp_rw [ENNReal.coe_le_coe, ← NNReal.mul_rpow, NNReal.rpow_le_rpow_iff hp]
   exact h
 
+-- XXX: should the nnnorm version be deleted or primed instead?
```

Comment:
The `nnnorm` versions should eventually get deprecated/deleted, but that can be a later PR.